sklearn/svm/base.py

```diff
@@ -779,6 +779,12 @@ def _fit_liblinear(X, y, C, fit_intercept, intercept_scaling, class_weight,
     if verbose:
         print('[LibLinear]', end='')
 
+    # LinearSVC breaks when intercept_scaling is 0
```

Comment:
I may be carping, but the origin of a problem is in the way liblinear handles bias, so I'd rather say that liblinear doesn't support negative values.